strawberry_django/filters.py

```diff
@@ -0,0 +1,140 @@
+import functools
+from typing import Type, Optional, List
+import strawberry
+from graphql import GraphQLError
+from strawberry.arguments import UNSET, is_unset
+
+
+__all__ = [
+    "FilterSetError",
+    "apply_filter",
+    "filter",
+    "get_filter_field_type",
+    "set_filter_field_type",
+]
+
+try:
+    import django_filters
+except ModuleNotFoundError:
+    class DummyDjangoFilters:
+        def __getattribute__(self, attr):
+            return attr
+
+    django_filters = DummyDjangoFilters()
+
+
+def assert_django_filters_installed(fn):
+    @functools.wraps(fn)
+    def wrapper(*args, **kwargs):
+        try:
+            import django_filters
+        except ModuleNotFoundError:
+            raise ModuleNotFoundError(
+                'You need to install django-filters to use "strawberry_django.filters". '
```

Comment:
I just noticed that the name of this package in pypi is `django-filter`, not `django-filters`. Should we use `django-filter` name instead?

Reply:
Yeah good idea

Reply:
done! thanks